pulseapi/users/signals.py

```diff
@@ -1,21 +1,12 @@
-from django.conf import settings
 from django.dispatch import receiver
-from django.db.models.signals import pre_save, post_delete
+from django.db.models.signals import post_delete
 
 from .models import EmailUser
 from pulseapi.profiles.models import UserProfile
 
-
-@receiver(pre_save, sender=EmailUser)
-def default_to_non_active(sender, instance, **kwargs):
-    if settings.TESTING:
-        return
-    if instance._state.adding is True:
-        instance.is_active = False
-
-
 @receiver(post_delete, sender=EmailUser)
 def delete_profile_for_user(sender, **kwargs):
-    related_profile_id = kwargs['instance'].profile_id
-    related_profile = UserProfile.objects.get(id=related_profile_id)
-    related_profile.delete()
+    if kwargs['instance'].profile_id:
```

Comment:
This if statement is to fix a bug that is occurring whenever I am trying to delete a account without a linked profile. 

Creating a ticket to investigate what is causing these "orphan profiles" in the first place.

Reply:
**Question** Is that really an issue with "orphan profiles"? It sounds more like users without profiles.

Reply:
Maybe these users are created in an alternative paths that is not using the `EmailUserManage.create_user` method?